templates/severity_inconsistency_needinfo.txt

```diff
@@ -0,0 +1,4 @@
+The severity field for this bug is set for to {{ extra[bugid]["severity"] }}, however, Whiteboard shows a higher severity with the flag {{ extra[bugid]["whiteboard_severity"] }}.
+:{{ nickname }}, could you have a look please?
+
+{{ documentation }}
```

Comment:
CC @eeejay @jcsteh @MReschenberg, in case you have any preference for the wording of the needinfo

Reply:
This looks good to me :)